packs/data/feats.db/chronoskimmer-dedication.json

```diff
@@ -24,7 +24,65 @@
         "prerequisites": {
             "value": []
         },
-        "rules": [],
+        "rules": [
+            {
+                "domain": "initiative",
+                "key": "RollOption",
+                "label": "Chronoskimmer Stabilize",
```

Comment:
At the very least we need to localize these, but I'm not keen on having 3 toggles on the sheet for something that can safely live without a toggle like assurance gets.